tools/maplint/lints/multiple_computers.yml

```diff
@@ -0,0 +1,4 @@
+/obj/machinery/computer:
```

Comment:
Is there any instance where we want several ``/obj/machinery`` stacked ontop of eachother?

Reply:
Applied with 57a3784a354cf7438d018e395ad5905204759186